README.md

```diff
@@ -5,6 +5,22 @@ This         is          the         source          code         behind
 [raw.githack.com](https://raw.githack.com)  - CDN  for your  source code
 that serves files with proper `Content-Type` headers.
 
+Supported Platforms
+-------------------
+
+raw.githack.com supports the following source code hosting platforms:
+
+- **[GitHub](https://github.com)** - served via `raw.githack.com` subdomain
+- **[GitHub Gist](https://gist.github.com)** - served via `gist.githack.com` subdomain
+- **[GitLab](https://gitlab.com)** - served via `gl.githack.com` subdomain
+- **[Bitbucket](https://bitbucket.org)** - served via `bb.githack.com` subdomain
+- **[Gitea](https://gitea.com)** - served via `gitea.githack.com` subdomain
+- **[Codeberg](https://codeberg.org)** - served via `codeberg.githack.com` subdomain
+- **[SourceHut Git](https://git.sr.ht)** - served via `srht.githack.com` subdomain
+- **[SourceHut Mercurial](https://hg.sr.ht)** - served via `srhgt.githack.com` subdomain
```

Comment:
Unfortunately, I had to remove this functionaly in https://github.com/neoascetic/rawgithack/commit/aea9677f7457a8111e60a66b7d72f9bacd735d35 as they started to block external requests heavily (to block AI bots crawling, but githack was also affected).

Reply:
Yes, please